### PR TITLE
(feat) Add problem response section

### PR DESCRIPTION
**Pull request by dsamojlenko**
# Summary | Résumé

Adds a link to view Responses in Problem status. The link appears below the table next to the "Report a problem link."

In future, the intention is to have a Problem filter at the top like the other statuses, but have it hidden unless there are problems.

image

### CODE REVIEW

**Comment by dsamojlenko** [comment]
Noting this comment left on the previously closed version of this change
image

..the paste function should now work correctly

File: public/static/locales/fr/form-builder-responses.json [hunk 1]
```diff
@@ -39,9 +44,13 @@
...
+      "deleted": "Pour suppression",
+      "problem": "Problème"
+    },
+    "viewAllProblemResponses": "Afficher toutes les réponses signalées avec problèmes"
   },
+  "supportWillContact": "L’équipe de soutien vous contactera",
+  "reportedAsProblem": "Réponse signalée avec problème",
   "downloadSuccess": {
     "title": "Réponses déplacées vers Téléchargements",
     "body": "Prochaine étape : autorisez la suppression des réponses de Formulaires GC."
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder-responses.json:56]:
```suggestion
    "body": "Prochaine étape : approuvez la suppression des réponses de Formulaires GC."
```

File: public/static/locales/fr/form-builder-responses.json [hunk 2]
```diff
@@ -91,7 +100,9 @@
       "noDownloadedResponses": "Aucune réponse téléchargée récemment",
       "noDownloadedResponsesMessage": "Les réponses téléchargées apparaîtront ici jusqu'à ce que vous approuviez leur suppression de Formulaires GC.",
       "noDeletedResponses": "Aucune réponse à supprimer",
-      "noDeletedResponsesMessage": "Une fois que vous aurez téléchargé et approuvé la suppression des réponses, celles-ci seront disponibles ici pendant 30 jours."
+      "noDeletedResponsesMessage": "Une fois que vous aurez téléchargé et approuvé la suppression des réponses, celles-ci seront disponibles ici pendant 30 jours.",
+      "noProblemResponses": "[FR] No problem responses",
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder-responses.json:104]:
```suggestion
      "noDeletedResponsesMessage": "Une fois que vous aurez téléchargé et approuvé la suppression des réponses, celles-ci seront disponibles ici pendant 30 jours.",
      "noProblemResponses": "Aucune réponse signalée avec problème",
```

File: lib/clientHelpers.ts [hunk 3]
```diff
@@ -356,7 +356,7 @@ export const ucfirst = (string: string) => {
 export const isStatus = (query: string, status: VaultStatus | VaultStatus[]): boolean => {
   const ucQuery = ucfirst(query);
   if (Array.isArray(status)) {
-    status.includes(ucQuery as VaultStatus);
+    return status.includes(ucQuery as VaultStatus);
```
Comment (thiessenp-cds) [inline lib/clientHelpers.ts:359]: Good find :)

**Review by thiessenp-cds — APPROVED** [review]
Works as expected. Nice work! LGTM :)

**Pull request merged**